lampstackplusdev/Dockerfile

```diff
@@ -69,15 +72,4 @@ RUN /usr/sbin/usermod -s /bin/zsh app && \
 #USER app
 #CMD ["/bin/zsh"]
 
-### /James Developer ---------------------------------------------------------
-### Management ---------------------------------------------------------------
-
-# This is the ending of the fullstack image, but we add it again here
-# to make it clear that this image is managed using supervisor
-
-USER root
-
-# Command that will run when the server starts
-CMD ["/usr/bin/supervisord", "--nodaemon", "--configuration", "/etc/supervisord.conf"]
-
-### /Management --------------------------------------------------------------
+### /James Developer ---------------------------------------------------------
```

Comment:
The Management section was a redundant repeat of the CMD/ENTRYPOINT declaration from the parent class